tests/unit/SbcFeeSummary.spec.ts

```diff
@@ -0,0 +1,372 @@
+import Vue from 'vue'
```

Comment:
This really should be in sbc-common-components repo (and combined with the suite there, which is pretty thin). Let me know if you feel strongly about this.